client/src/components/BulkCreateForm/BulkCreateForm.js

```diff
@@ -219,20 +219,16 @@ export default class BulkCreateForm extends React.Component {
         });
     }
 
-    downloadCSV(results) {
-        const fields = ['Row Number', 'Warnings', 'Errors'];
-        results = results.map(result => {return {
+    buildCsvExportFields() {
+        return ['Row Number', 'Warnings', 'Errors'];
+    }
```

Comment:
You could just make this a property

```javascript
class BulkCreateForm {
    
    csvExportFields = ['Row Number', 'Warnings', 'Errors'];
}
```
or just make it a constant outside the class
```javascript
const csvExportFields = ['Row Number', 'Warnings', 'Errors'];

class BulkCreateForm {

}

Reply:
I made it a property.